src/port1.0/portsandbox.tcl

```diff
@@ -142,4 +155,10 @@ proc portsandbox::set_profile {target} {
             }
         }
     }
+
+    if {${build.type} eq "xcode"} {
+        # let Xcode create jobs (FIXME: narrow allowed commands)
+        append portsandbox_profile "\
+(allow job-creation)"
```

Comment:
Do you know if this syntax works on all macOS versions we run on?

Reply:
The buildbot tests should tell us if it causes errors on 10.6. If it works there I'll make a note to check 10.5 before releasing this.

Reply:
User reported this fails on Mac OS X 10.5: https://trac.macports.org/ticket/72940